source/jormungandr/jormungandr/realtime_schedule/realtime_proxy_manager.py

```diff
@@ -56,6 +57,7 @@ def __init__(self, proxies_configuration):
                 log.warn('impossible to build a realtime proxy, missing mandatory field in configuration')
                 continue
             object_id_tag = configuration.get('object_id_tag', proxy_id)
+            destination_id_tag = configuration.get('destination_code_type', None)
```

Comment:
I don't think you need this line, it will be automatically be read from the configuration and given to the `**kwargs`